<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ques_no_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_left_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="marks_obtained_label">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="home_button">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="question_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answer_textbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="previous_ques_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_ques_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="begin_button">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Begin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submit_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
